tests/E2E/package.json

```diff
@@ -27,6 +27,7 @@
   },
   "homepage": "https://github.com/PrestaShop/PrestaShop#readme",
   "dependencies": {
+    "got": "^9.6.0",
```

Comment:
why not using chai-http?

Reply:
I didn't know about chai-http.
Thanks for suggestion.